index.html

```diff
@@ -1,10 +1,166 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Title</title>
-</head>
-<body>
-
-</body>
+<!doctype html>
+<html lang="pt-br">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="assets/css/main.css">
+    <!-- Fonts -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,700;1,700&display=swap"
+      rel="stylesheet">
+    <title>Appoint | Marque a sua consulta</title>
+  </head>
+
+  <body>
+    <!-- Navbar -->
+    <header id="header">
+      <div class="container-fluid">
+        <nav class="navbar navbar-expand-md navbar-light">
+          <a class="navbar-brand" href="#">
+            <img src="assets/img/logo.svg" height="36">
+          </a>
+
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav"
+            aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+            <!--<span class="navbar-toggler-icon">Menu</span> -->
+            Menu
+          </button>
+
+          <div class="collapse navbar-collapse justify-content-end" id="navbarNav">
+            <ul class="navbar-nav">
+              <li class="nav-link">
+                <a class="nav-link active " aria-current="page" href="#"
+                  style="font-size: 12px; line-height: 15px; color: var(--primary); margin-top: 5px;">AJUDA</a>
```

Comment:
Remover todos os in-line styles para criar classes mais robustas no css.

Reply:
Feito